webviz_subsurface/plugins/_tornado_plotter_fmu/shared_settings/_view_settings.py

```diff
@@ -0,0 +1,131 @@
+from typing import List
+
+import pandas as pd
+import webviz_core_components as wcc
+from dash import Input, Output, callback, html
+from dash.development.base_component import Component
+from webviz_config.utils import StrEnum, callback_typecheck
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC
+
+
+class ViewSettings(SettingsGroupABC):
+    class IDs(StrEnum):
+        REFERENCE = "reference"
+        SCALE = "scale"
+        SENSITIVITIES = "sensitivities"
+        RESET_BUTTON = "reset-button"
+        PLOT_OPTIONS = "plot-options"
+        LABEL = "label"
+
+    def __init__(
+        self,
+        realizations: pd.DataFrame,
+        reference: str = "rms_seed",
+        allow_click: bool = False,
+    ) -> None:
+        super().__init__("View Settings")
+        unique_realizations = realizations["SENSNAME"].unique()
+        self.sensnames = list(unique_realizations)
+        self.scales = [
+            "Relative value (%)",
+            "Relative value",
+            "True value",
+        ]
+        self.plot_options = [
+            {
+                "label": "Remove sensitivites with no impact",
+                "value": "Remove sensitivites with no impact",
```

Comment:
Share settings - for both table and plot. Move out from list of plot options and create a single checkbox for true/false state.

Remember to update the callbacks in `_plugin.py`: `_calc_tornado_table()` and `_calc_tornado_plot()` with new element and value.